app/serializers/user_serializer.rb

```diff
@@ -5,9 +5,18 @@ class UserSerializer < ActiveModel::Serializer
   def home_duration_in_seconds
     active_session = object.active_session
     if active_session.present? && active_session.home?
-      object.home_duration_in_seconds + active_session.in_seconds
+      object.home_duration_in_seconds + Sessions::Duration.in_seconds(active_session)
     else
       object.home_duration_in_seconds
     end
   end
+
+  def wallet_balance
+    active_session = object.active_session
+    if active_session.present? && active_session.home?
```

Comment:
I think we should fetch the latest wallet balance even if the current session is an away session ?

Reply:
@arunava59 ah yeah, will change. Thanks for pointing out